schemas/root.schema.json

```diff
@@ -0,0 +1,183 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "required": ["signatures", "signed"],
+  "additionalProperties": false,
+  "properties": {
+    "signatures": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": ["keyid", "sig"],
+        "additionalProperties": false,
+        "properties": {
+          "keyid": {
+            "type": "string",
+            "minLength": 64,
+            "maxLength": 64,
+            "pattern": "^[a-f0-9]{64}$"
+          },
+          "sig": {
+            "type": "string",
+            "minLength": 1024,
+            "maxLength": 1024,
+            "pattern": "^[a-f0-9]{1024}$"
```

Comment:
One more thing to clarify - do we want to relax requirements on the signature? The spec does not talk about its length.

Reply:
Yes: should just be a positive length